www/kanban/inner.js

```diff
@@ -211,22 +212,26 @@ define([
                 verbose("in color click");
                 var board = $(el.parentNode).attr("data-id");
                 var boardJSON = kanban.getBoardJSON(board);
+                var onchange = function (colorL) {
+                    var elL = el;
+                    var boardL = $(elL.parentNode).attr("data-id");
+                    var boardJSONL = kanban.getBoardJSON(boardL);
+                    var currentColor = boardJSONL.color;
+                    verbose("Current color " + currentColor);
+                    if (currentColor !== colorL.toString()) {
+                        $(elL).removeClass("kanban-header-" + currentColor);
+                        boardJSONL.color = colorL.toString();
+                        kanban.onChange();
+                    }
+                };
+                var jscolorL;
+                el._jscLinkedInstance = undefined;
+                jscolorL = new jscolor(el,{onFineChange: onchange, valueElement:undefined});
+                jscolorL.show();
                 var currentColor = boardJSON.color;
-                verbose("Current color " + currentColor);
-                var index = kanban.options.colors.findIndex(function (element) {
-                    return (element === currentColor);
-                }) + 1;
-                verbose("Next index " + index);
-                if (index >= kanban.options.colors.length) { index = 0; }
-                var nextColor = kanban.options.colors[index];
-                verbose("Next color " + nextColor);
-                boardJSON.color = nextColor;
-                $(el).removeClass("kanban-header-" + currentColor);
-                $(el).addClass("kanban-header-" + nextColor);
-                kanban.onChange();
+                jscolorL.fromString(currentColor);
             },
-            buttonClick: function (el, boardId, e) {
-                e.stopPropagation();
+            buttonClick: function (el, boardId) {
```

Comment:
what are the implications of removing the `stopPropagation` call? I don't see how that relates to the color picker.

Reply:
I dont think I did that intentional. I will recheck and readd if it hs no change to what I added